spec/functional/resource/registry_spec.rb

```diff
@@ -40,18 +40,42 @@
   end
 end
 
-describe Chef::Resource::RegistryKey, :windows_only, broken: true do
+describe Chef::Resource::RegistryKey, :windows_only do
+  class ::Chef::Resource::RegistryKey
+    # for test purposes only, don't scrub the values
+    def scrub_values(values)
+      values
+    end
+  end
+
+  instance_eval do
+    def context_let(symbol, &block)
```

Comment:
the "`let`"s used in this test are called both inside and outside of examples and thus, they need to be visible with the same value for both cases.